presto-native-execution/scripts/dockerfiles/README.md

```diff
@@ -0,0 +1,54 @@
+# Prestissimo - Dockerfile build
+
+> 📝 _**Note:** Please post in Presto Slack if you have any questions_
+
+There are two kinds of dockerfiles: 
+1) A platform specific file to build dependencies. Current supported platforms are
+   1) Centos-8-stream with gcc9.
+   2) Ubuntu-22.04 with gcc11.
+2) A platform agnostic file to build Prestissimo runtime on top of the dependency image.
+### Dependency Dockerfiles
+These dockerfile installs all the dependencies including those needed for testing.
+A list of dependencies can be found [here](../../README.md).
+By default, the dependencies are built in Release mode.
+This image needs to be built only when some dependency is updated.
```

Comment:
```suggestion
The Dependency Image needs to be built only when some dependency is updated.
```
You refer to "Dependency Image" further down in the Quick Start. It's a good phrase, and introducing it here can help connect this explanation to the steps in the Quick Start.